js/kraken.js

```diff
@@ -971,23 +971,7 @@ module.exports = class kraken extends Exchange {
         }
         return {
             'id': id,
-            'clientOrderId': clientOrderId,
             'info': response,
-            'timestamp': undefined,
-            'datetime': undefined,
-            'lastTradeTimestamp': undefined,
-            'symbol': symbol,
```

Comment:
I think `symbol` is ok to keep defined. @npomfret what do you think?

Reply:
That's a good point, symbol is fine